frontend/src/pages/EventCreate.tsx

```diff
@@ -25,6 +27,14 @@ export default function EventCreate() {
       addToastError(t("required") + ": " + t("uploadImage"), 400);
       return;
     }
```

Comment:
Create a now variable of value dayjs() that way your not recreating the the now time per if statement.